editor/types/peopleList.ts

```diff
@@ -7,16 +7,16 @@ export type PeopleListRow = {
   crash_pk: number;
   crash_timestamp: string;
   drvr_city_name: string;
-  drvr_ethncty: LookupTableOption;
-  prsn_ethnicity_id: number;
+  drvr_ethncty?: LookupTableOption;
```

Comment:
Glad to see you catching this typeError waiting to happen!

I haven't written an issue for this, but I'd really like to go through all our types and make every property that comes from Hasura optional (`?`) 😅